packages/graphql-migrations/src/util/defaultNameTransforms.ts

```diff
@@ -0,0 +1,15 @@
+import { NameTransformDirection } from '../abstract/generateAbstractDatabase'
+
+export function defaultColumnNameTransform(name: string, direction: NameTransformDirection) {
+  if (direction === 'to-db') {
+    return `${name.charAt(0).toLowerCase()}${name.slice(1)}`
```

Comment:
Why we need that?

What will happen if schema will contain fields like this:

```
type User {
   USER_NAME: String
   USER_PASSWORD: String
   DATA_VALIDATED: Boolean
}
```

Reply:
I think we simply my want to revert that and accept field names as they are?

Reply:
You're right. Will need to also check how the code-generator does it.